.github/scripts/install-deps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+distro=$1
+distro_ver=$2
+
+# install required packages
+if [ "$distro" == "ubuntu" ]; then
+    apt update -y
+    apt install -y git wget
+
+    # if running on xenial, install newer python version from ppa
```

Comment:
This is needed for `tuttest` right? If so, can you be clear that is the case and this isn't used by the toolchain at all?

Reply:
I extracted the tuttest from the docker script. So this part is no longer needed.